renderers/photoaudiohaptics.schema.json

```diff
@@ -25,12 +25,16 @@
                 "maximum": 1
             }
         },
-        "contourPoints": {
+        "contours": {
```

Comment:
To clarify, this is all the points with a label and not broken down by individual contours, right?

Reply:
yes